include/pcmki/pcmki_rule.h

```diff
@@ -0,0 +1,31 @@
+#ifndef PCMK__PCMKI_PCMKI_RULE__H
```

Comment:
Remember the copyright boilerplate up here, too.

Reply:
Thanks! I copied `include/pcmki/pcmki_cluster_queries.h` to use as my template, and it doesn't have copyright boilerplate. I'll add that before the next push.